### DISCUSSION TITLE
Welcome to TrAdaboost Discussions!

### DISCUSSION THREAD

**Discussion by Bin-Cao** [Bin-Cao/TrAdaboost#4] [Announcements]
Bin : discussions are welcome here

**Comment by colayuuuu** [comment]
As for a multi output regression problem(multi input- multi output), there is some key points to modify the algoruthm(e.g. TrAdaboost.T2)?

**Reply by Bin-Cao** [reply]
In the multi-output scenario, the adjusted error e_ti for each instance can be accurately calculated. This adjustment can be achieved by treating the yj and ht(xj) as vectors and calculating the norm of the difference vector.
